Dockerfile.builder

```diff
@@ -33,6 +33,11 @@ RUN apt-get update && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+#
+# Retrieve llvm-objcopy binary
+#
+COPY --from=docker.io/cilium/cilium-llvm:cae23fe2f43497ae268bd8ec186930bc5f32afac /usr/local/bin/llvm-objcopy /bin/
```

Comment:
nit: is it possible to make the image an `ARG` here as done in `images/cilium/Dockerfile` below?

Specifically:
* Add `ARG CILIUM_LLVM_IMAGE=docker.io/cilium/cilium-llvm:cae23fe2f43497ae268bd8ec186930bc5f32afac` towards the top of the file.
* Write this line as `COPY --from=$CILIUM_LLVM_IMAGE /usr/local/bin/llvm-objcopy /bin/`

This avoids having a hardcoded magic number (i.e. the SHA256 hash) in the code and both adds meaning to the number and makes it easier to change in the future.

Reply:
Oh, nice! I didn't know about that.

Though, Docker doesn't seem to like in this case (no pun intended):
```
Step 11/12 : COPY --from=${CILIUM_LLVM_IMAGE} /usr/local/bin/llvm-objcopy /bin/
invalid from flag value ${CILIUM_LLVM_IMAGE}: invalid reference format: repository name must be lowercase
```
I'm guessing that's why we don't do that for any other image we have.